### ISSUE TITLE
Problem after updating React to 0.13: 

### ISSUE THREAD

**Issue by mg** [fraserxu/react-chartist#8]
Hi,

just updated React to 0.13 and now, when React-Chartist runs the updateChart() function, specifically the 
```
this.chartist = new Chartist[type](this.getDOMNode(), data, options, responsiveOptions);
```
React bombs with
```
Cannot read property 'firstChild' of undefined
```
Has anyone else tried updating?

Cheers,
mg

**Comment by fraserxu** [comment]
Thanks for pointing it out. I haven't played with react 0.13 yet, I'll check and see if I can fix.

**Comment by fraserxu** [comment]
I checked the v0.13 release note and find this:

> Added new top-level API React.findDOMNode(component), which should be used in place of component.getDOMNode(). The base class for ES6-based components will not have getDOMNode. This change will enable some more patterns moving forward.

Will test it out.

**Comment by fraserxu** [comment]
Hi @mg I pushed the changes here https://github.com/fraserxu/react-chartist/commit/b7832a1c2a7c3e47fe6910c08275a7b1e3c981ea with react v0.13.1 and also bump to chartist version to v0.7.3.

I checked and tested the code in the example folder. Now just published it in react-chartist@0.4.0

Can you try it? Thanks.

**Comment by mg** [comment]
Works! Thanks for the quick response.
